### PR TITLE
afl++ tests

### PR DESCRIPTION
**Pull request by vanhauser-thc**
First benchmark with libafl! @domenukk @andreafioraldi 
tests libafl in bug benchmark and normal
tests frida fix for normal
tests cnt variant on bug

the curl benchmark fails for libafl at "make check", but the output shows that the fuzzer builds fine, I think make check just expects something that is different how the libafl is behaving

### CODE REVIEW

**Comment by vanhauser-thc** [comment]
that is the curl error output, it shows that the fuzzer is ready
```
#18 318.8 + make check
#18 318.8 error: Found argument '/src/curl_fuzzer/corpora/curl_fuzzer_dict/test_
wrongproto' which wasn't expected, or isn't valid in this context
#18 318.8 
#18 318.8 If you tried to supply `/src/curl_fuzzer/corpora/curl_fuzzer_dict/test
_wrongproto` as a PATTERN use `-- /src/curl_fuzzer/corpora/curl_fuzzer_dict/test
_wrongproto`
#18 318.8 
#18 318.8 USAGE:
#18 318.8     curl_fuzzer_dict [OPTIONS] <out> <in>
#18 318.8 
#18 318.8 For more information try --help
```

**Comment by vanhauser-thc** [comment]
libafl will need another iteration to get working for all targets, removing it as Dominik is on vacation now ...

File: fuzzers/libafl/fuzzer.py [hunk 1]
```diff
@@ -11,33 +11,32 @@
...
-    os.environ['AFL_FAST_CAL'] = '1'
-    os.environ['AFL_CMPLOG_ONLY_NEW'] = '1'
-
-    aflplusplus_fuzzer.fuzz(input_corpus,
-                            output_corpus,
-                            target_binary,
-                            flags=(run_options))
+    afl_fuzzer.prepare_fuzz_environment(input_corpus)
+    dictionary_path = utils.get_dictionary_path(target_binary)
+    command = ([target_binary])
```
Comment (jonathanmetzman) [inline fuzzers/libafl/fuzzer.py:37]: nit: paren not needed.

**Review by jonathanmetzman — COMMENTED** [review]
Can you add libafl to CI (fuzzers.yml)

File: fuzzers/libafl/fuzzer.py (hunk 1 above)
Comment (vanhauser-thc) [reply fuzzers/libafl/fuzzer.py]: just removed libafl, it needs some more rework and dominik is now gone for the weekend.

**Comment by jonathanmetzman** [comment]
> Can you add libafl to CI (fuzzers.yml)

Sorry, didn't notice your above comment.

**Comment by domenukk** [comment]
Will happen soon though :)

**Pull request merged**